src/main/java/io/supertokens/emailpassword/PasswordHashingUtils.java

```diff
@@ -70,26 +102,15 @@ public static String updatePasswordHashWithPrefixIfRequired(String passwordHash,
             return passwordHash;
         }
 
-//        if (hashingAlgorithm == PasswordHashingAlgorithm.SCRYPT){
-//            if (doesPasswordHashHaveScrpytPrefix(passwordHash)){
-//                return passwordHash;
-//            }
-//            return addScryptPrefixToPasswordHash(passwordHash);
-//        }
-
+        if (hashingAlgorithm == CoreConfig.PASSWORD_HASHING_ALG.FIREBASE_SCRYPT) {
```

Comment:
im questioning if it still makes sense to have this function - since we will be telling users to give the hashes in a specific format anyway, so we can ask them to add the desired prefix as well